src/modules/75-cf/components/DetailPageTemplate/DetailPageTemplate.tsx

```diff
@@ -29,49 +30,59 @@ export interface DetailPageTemplateProps {
   /** Optional extra components to be added into header (context menu, edit button, etc...)  */
   /** Note: Use absolute position to style it */
   headerExtras?: React.ReactNode
+  menuItems?: React.ReactNode
+  metaData?: Record<string, string>
 }
 
 export const DetailPageTemplate: React.FC<DetailPageTemplateProps> = ({
   breadcrumbs,
   title,
   subTitle,
   identifier,
-  headerExtras,
-  children
+  children,
+  menuItems,
+  metaData
 }) => {
+  const { getString } = useStrings()
+
   return (
     <>
-      <Container
-        height={HEADER_HEIGHT}
-        padding={{ top: 'large', right: 'xlarge', bottom: 'large', left: 'xlarge' }}
-        style={{
-          background: 'rgba(219, 241, 255, .46)',
-          position: 'relative',
-          borderBottom: '.5px solid rgb(227 224 224 / 50%)'
-        }}
-      >
-        <Layout.Vertical spacing="medium">
+      <Page.Header
+        size="large"
+        breadcrumbs={
           <NGBreadcrumbs
             customPathParams={{ module: 'cf' }}
             links={breadcrumbs.map(({ title: label, url }) => ({ label, url }))}
           />
-          <Container>
-            <Layout.Horizontal spacing="medium">
-              <Text style={{ fontSize: '20px', color: 'var(--black)' }}>{title}</Text>
-              {identifier && <IdentifierText identifier={identifier} allowCopy style={{ marginBottom: 0 }} />}
+        }
+        title={
+          <>
+            <Layout.Horizontal spacing="small">
+              <Heading level={3} font={{ variation: FontVariation.H4 }}>
+                {title}
+              </Heading>
+              {identifier && <IdentifierText identifier={identifier} allowCopy />}
             </Layout.Horizontal>
             {subTitle && (
               <Text color={Color.GREY_400} padding={{ top: 'xsmall' }}>
                 {subTitle}
               </Text>
             )}
-          </Container>
-        </Layout.Vertical>
-        {headerExtras}
-      </Container>
-      <Container style={{ height: `calc(100% - ${HEADER_HEIGHT}px)`, overflow: 'auto', background: '#e4ebf433' }}>
-        {children}
-      </Container>
+          </>
+        }
+        content={
+          <>
+            {menuItems && menuItems}
```

Comment:
And then here you can check if there are items and output the menu (positioned correctly).
```suggestion
            {!!menuItems.length && <RbacOptionsMenuButton items={menuItems} />}
```

Reply:
done